src/org/netbeans/modules/php/wordpress/resources/templates/package-info.java

```diff
@@ -7,35 +7,40 @@
             description = "WpGPLv2LicenseDescription.html",
             scriptEngine = "freemarker"),
     @TemplateRegistration(
-            folder = "WordPress",
+            folder = "Scripting/WordPress",
```

Comment:
I prefer "WordPress". (Please don't change it.)

Reply:
I would say this has less to do with preferences. WordPress belongs to PHP. We also have it inside the global options -> PHP -> Frameworks & Tools and also inside the project properties -> Frameworks. At the end I don't care about if you really want to change this back but as I said, it belongs to it and should be there too.

Reply:
Category is already PHP by the your below change, right?
If we add the folder, we have to click the folder once every time. I don't want to do it.

Reply:
Yes, you are right, it is already for this category. I will change the other thing back. Thx for your info.

Reply:
Fixed

Reply:
OK.